app/src/pages/CreateProject.svelte

```diff
@@ -0,0 +1,364 @@
+<script>
```

Comment:
This is getting quite large, can we capture a follow-up where we break it apart in components? Or would that be not warranted?

Reply:
I would love to!
https://linear.app/issue/MON-38